packages/dds/sequence/src/intervals/intervalUtils.ts

```diff
@@ -183,6 +172,10 @@ export interface ISerializableInterval extends IInterval {
 	getIntervalId(): string;
 }
 
+export type ISerializableIntervalPrivate<T extends ISerializableInterval> = T & {
+	propertyManager?: PropertiesManager;
```

Comment:
both `SequenceInterval` and `Interval` initialize their property manager on creation rather than lazily now, and this interface is an implementation detail, so might as well go with the stricter type
```suggestion
	propertyManager: PropertiesManager;
```

Reply:
This causes the change to sprawl a ton, as we lose the implicit conversion between ISerializableInterval and ISerializableIntervalPrivate, so i'd rather not do it here to keep the change a bit more contained. Overall, there is a lot of cleanup necessary around shared intervals that i don't want to take on completely here

Reply:
ah, yeah. that's fine then, can always do in a follow-up.